src/tests/test_run.py

```diff
@@ -78,9 +78,63 @@ def PolarsDataFrame(cls):
         def fetch_results(self, fetch_all=False):
             pass
 
+        @classmethod
+        def pie(self, key_word_sep=" ", title=None, **kwargs):
+            warnings.warn(
+                (
+                    ".pie() will be removed in a future version,"
+                    "please use %sqlplot pie. If you need help migrating,"
+                    "send us a message: https://ploomber.io/community"
+                )
+            )
+
+        @classmethod
+        def bar(self, key_word_sep=" ", title=None, **kwargs):
+            warnings.warn(
+                (
+                    ".bar() will be removed in a future version,"
+                    "please use %sqlplot bar. If you need help migrating,"
+                    "send us a message: https://ploomber.io/community"
+                )
+            )
+
+        @classmethod
+        def plot(self, key_word_sep=" ", title=None, **kwargs):
+            warnings.warn(
+                (
+                    ".plot() will be removed in a future version,"
+                    "If you need help migrating,"
+                    "send us a message: https://ploomber.io/community"
+                )
+            )
+
     return ResultSet
 
 
+def test_pie_warning(mock_resultset):
```

Comment:
use pytest parametrize feature to run three separate tests, instead of calling all the functions in the same test

Reply:
Update the test